aggregator_core/src/datastore.rs

```diff
@@ -128,9 +129,17 @@ impl<C: Clock> Datastore<C> {
         crypter: Crypter,
         clock: C,
         meter: &Meter,
+        max_transaction_retries: u64,
     ) -> Result<Datastore<C>, Error> {
-        Self::new_with_supported_versions(pool, crypter, clock, meter, SUPPORTED_SCHEMA_VERSIONS)
-            .await
+        Self::new_with_supported_versions(
```

Comment:
At some point we'll want to use the builder pattern here instead of tacking on more constructor arguments.

Reply:
Indeed, I think we've reached the limit--any more we'll want to move to builder pattern.